contracts/ERC721Universal.sol

```diff
@@ -2,18 +2,18 @@
 pragma solidity ^0.8.20;
 
 import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
-import "./IERC721BridgelessMinting.sol";
+import "./IERC721Universal.sol";
```

Comment:
why not UniversalMinting ?  Universal is a very general word in my opinion

Reply:
The main reason is that it is both for minting and evolving. So this is one of those cases where less is more. I've made sure that the evolution part is also explicitly stated everywhere in the new commits